opencart 1.5.6.4/upload/admin/language/english/module/smaily_for_opencart.php

```diff
@@ -62,3 +62,14 @@
 // RSS text
 $_['rss_feed_title'] = 'Product RSS feed';
 $_['rss_feed_text'] = "Copy this URL into your template editor's RSS block, to receive RSS-feed";
+
+// Layout texts
+$_['entry_layout'] = 'Layout:';
```

Comment:
Are `entry_...` phrases needed here? Shouldn't OpenCart provide them out of the box?

Reply:
funnily enough yes, every stock plugin has the same `entry_` phrases in their language files.